client/components/housing/HouseCard.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import styled from 'styled-components'
 import { useRouter } from 'next/router'
 import { TAGS, TagGroup, TagRow, TagBubble } from './Tag'
```

Comment:
If we'll be using tags in the future, can we wait until #138 is in before we merge this? The persisting tech debt in this branch is somewhat uncomfortable, and tags is a big part of it

Reply:
Merged in #138